src/c_kzg_4844.h

```diff
@@ -159,6 +93,10 @@ typedef enum {
  * Stores the setup and parameters needed for computing KZG proofs.
  */
 typedef struct {
+    /** Number of field elements in a blob. */
+    uint64_t field_elements_per_blob;
+    /** Number of bytes in a blob. */
```

Comment:
Does it make sense to have an additional field for `bytes_per_blob`? we can just do `field_elements_per_blob * BYTES_PER_FIELD_ELEMENT` wherever required right? Or define a function that does that

Reply:
I'm open to removing it. I thought it was a nice convenience field; it's used several times. I don't think a function would be appropriate.

Reply:
IMO it's OK to keep `bytes_per_blob` there. We currently have a `BYTES_PER_BLOB` macro, so it keeps the same logic as we have currently.

Not super strong opinion here. We could also remove it, if people insist on it.